app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+# Create user model
+class User < ApplicationRecord
+  has_many :posts, class_name: 'Post'
+  has_many :comments, class_name: 'Comment'
+  has_many :likes, class_name: 'Like'
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ ] Following the project requirements, this method should return the 3 most recent posts. Kindly update that.